app/graphql/types/query_type.rb

```diff
@@ -14,12 +14,21 @@ class QueryType < Types::BaseObject
     #   'Hello World - This is your Interview Challenge!\nGood luck.'
     # end
 
-    field :getAllCandidates, [Types::CandidateType], null: false do
+    field :allCandidates, [Types::CandidateType], null: false do
       description "This will get all the candidates."
     end
 
-    def getAllCandidates
+    def allCandidates
       return Candidate.all
     end
+
+    field :candidate, Types::CandidateType, null: false do
```

Comment:
Added this new field for a specific candidate. It was a little tricky because I did not realize I had to take the CandidateType out of the brackets because I was no longer looking for an array of candidates, only one.